src/Nethermind/Nethermind.Network/Metrics.cs

```diff
@@ -61,107 +61,107 @@ public static class Metrics
         public static long LesStatusesSent { get; set; }
 
         [CounterMetric]
-        [Description("Number of received disconnects due to breach of protocol")]
+        [Description("_Deprecated._ Number of received disconnects due to breach of protocol")]
```

Comment:
Why are they now deprecated, can we remove them?

Reply:
Can't remove them now. Need to mark them as deprecated first, them remove them after some release.

Reply:
Otherwise people who currently use them suddenly would have missing metrics.

Reply:
So why are we deprecating them?

Reply:
Because they all can be replaced with only two metric which is much easier to graph with label. So:
1. Mark with deprecated.
2. Wait another release so that user can move to the two metric.
3. Remove deprecated metrics.